doc/release-notes/11.2.md

```diff
@@ -0,0 +1,39 @@
+# Release notes for Gluster 11.2
+
+This is a bugfix release. The release notes for [11.0](11.0.md) contains a listing of all the new improvements and bugs fixed in the GlusterFS 11 stable release.
```

Comment:
Need to update this line also I think @sanjurakonde with the 11.2 and 11.2.md file